src/table/components/Announce.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles({
+  screenReaderOnly: {
+    clip: 'rect(0 0 0 0)',
+    clipPath: 'inset(50%)',
+    height: '1px',
+    overflow: 'hidden',
+    position: 'absolute',
+    whiteSpace: 'nowrap',
+    width: '1px',
+  },
+});
+
+const Announce = () => {
```

Comment:
`Announcer` would make more sense

Reply:
Or something that is a not the same as the `announce` function

Reply:
I think `AnnounceElements` would fits well in this case, because `Announcer` refers to the functionality but these are just elements for announcing. so in the latest commit i will switch the name to "AnnounceElements"